lessons/models.py

```diff
@@ -62,6 +62,22 @@ class Meta:
         ordering = ["word"]
         verbose_name_plural = "vocab words"
 
+    @property
+    def i18n_key(self):
+        return slugify(self.get_untranslated_field('word'))
+
+    @classmethod
+    def internationalizable_fields(cls):
+        return ['word', 'simpleDef', 'detailDef']
+
+    @classmethod
+    def get_i18n_objects(cls):
+        return super(Vocab, cls).get_i18n_objects()
```

Comment:
note that it's only necessary to override this if you want to modify it, and the primary reason to modify it is to prefetch related models to make the `should_be_translated` fetches faster. For example:

https://github.com/mrjoshida/curriculumbuilder/blob/b04cec0692d213e7d32d9a7d18b16658c5aeca36/lessons/models.py#L112-L118

Reply:
Should I add a prefetch here, or ok to remove this entire class method?